packages/apt/test.sh

```diff
@@ -152,20 +152,23 @@ echo "::group::Upgrade"
 
 sudo apt purge -V -y ${pgroonga_package}
 
-# Disable upgrade test for first time packages.
-# TODO: Automate this. We don't want to maintain this manually.
-# case ${postgresql_version} in
-#   16) # TODO: Remove this after 3.1.4 release.
-#     exit
-#     ;;
-# esac
-
-sudo apt install -V -y ${pgroonga_package}
-createdb upgrade
-psql upgrade -c 'CREATE EXTENSION pgroonga'
-apt install -V -y \
-  ${repositories_dir}/${os}/pool/${code_name}/*/*/*/*_${architecture}.deb
-psql upgrade -c 'ALTER EXTENSION pgroonga UPDATE'
+if sudo apt install -V -y ${pgroonga_package}; then
+  # Disable upgrade test for first time packages.
+  # TODO: Automate this. We don't want to maintain this manually.
+  # case ${postgresql_version} in
+  #   16) # TODO: Remove this after 3.1.4 release.
+  #     exit
+  #     ;;
+  # esac
```

Comment:
Is it the same context about the unreleased package?
If so, I will delete this comment.

Reply:
Yes. (It's only for the unreleased package.)